### PR TITLE
Add NEXT_MAJOR comments

### PR DESCRIPTION
**Pull request by franmomu**
Some pedantic changes to ease resolution of changes in next major version.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -21,11 +21,11 @@
         <extension class="Symfony\Component\Panther\ServerExtension" />
     </extensions>
 
-    <filter>
-        <whitelist>
+    <coverage>
```
Comment (VincentLanglet) [inline phpunit.xml.dist:24]: Do you want to do the same PR on dev-kit ?

**Review by VincentLanglet — APPROVED** [review]

File: phpunit.xml.dist (hunk 1 above)
Comment (franmomu) [reply phpunit.xml.dist:24]: Yep, that was the idea.

**Comment by franmomu** [comment]
I've merged it since I think they are trivial changes and this allows me to work on make it compatible with SF5 and Admin 4.x

File: src/Model/ModelManager.php [hunk 2]
```diff
@@ -277,6 +277,13 @@ public function executeQuery($query)
             __METHOD__
         ), E_USER_DEPRECATED);
 
+        //throw new \InvalidArgumentException(sprintf(
```
Comment (phansys) [inline src/Model/ModelManager.php:280]: This should be `\TypeError`.

Comment (franmomu) [reply src/Model/ModelManager.php:280]: 👍 fixed in https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/481

**Pull request merged**